stage0/run.go

```diff
@@ -406,6 +430,10 @@ func Run(cfg RunConfig, dir string, dataDir string) {
 
 	destRootfs := common.Stage1RootfsPath(dir)
 
+	if len(cfg.DNS) > 0 || len(cfg.DNSSearch) > 0 || len(cfg.DNSOpt) > 0 {
```

Comment:
It's not our responsibility to do this nor can't we be certain that the application inside uses libc's resolver.